phpBB/styles/prosilver/template/overall_header.html

```diff
@@ -84,7 +84,7 @@
 
 			</div>
 		</div>
-
+<!-- EVENT overall_header_navbar_before -->
 		<!-- INCLUDE navbar_header.html -->
```

Comment:
They're not.
[https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/styles/prosilver/template/overall_header.html#L91](overall_header_page_body_before)
[https://github.com/phpbb/phpbb/blob/3.1.x/phpBB/styles/prosilver/template/overall_header.html#L103](overall_header_content_before)